TestCases/rans/flatplate/roughness/bc_wilcox2006/turb_SST_flatplate_roughBCWilcox2006.cfg

```
@@ -0,0 +1,248 @@
+MARKER_MONITORING= ( wall )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, LEAST_SQUARES, 
+%                                         WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+%
+% Courant-Friedrichs-Lewy condition of the finest grid
+CFL_NUMBER= 100.0
+%
+% Adaptive CFL number (NO, YES)
+CFL_ADAPT= YES
+%
+% Parameters of the adaptive CFL number (factor down, factor up, CFL min value,
+%                                        CFL max value )
+CFL_ADAPT_PARAM= ( 0.1, 2.0, 100.0, 1e5 )
+%
+% Runge-Kutta alpha coefficients
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
+%
+% Number of total iterations
+ITER= 99999
+
+% ----------------------- SLOPE LIMITER DEFINITION ----------------------------%
+%
+% Coefficient for the limiter
+VENKAT_LIMITER_COEFF= 0.1
+%
+% Coefficient for the sharp edges limiter
+ADJ_SHARP_LIMITER_COEFF= 3.0
+%
+% Reference coefficient (sensitivity) for detecting sharp edges.
+REF_SHARP_EDGES= 3.0
+%
+% Remove sharp edges from the sensitivity evaluation (NO, YES)
+SENS_REMOVE_SHARP= NO
+
+% -------------------------- MULTIGRID PARAMETERS -----------------------------%
+%
+% Multi-Grid Levels (0 = no multi-grid)
+MGLEVEL= 0
+%
+% Multi-grid cycle (V_CYCLE, W_CYCLE, FULLMG_CYCLE)
+MGCYCLE= V_CYCLE
+%
+% Multi-grid pre-smoothing level
+MG_PRE_SMOOTH= ( 1, 2, 3, 3 )
+%
+% Multi-grid post-smoothing level
+MG_POST_SMOOTH= ( 2, 2, 2, 2)
+%
+% Jacobi implicit smoothing of the correction
+MG_CORRECTION_SMOOTH= ( 0, 0, 0, 0 )
+%
+% Damping factor for the residual restriction
+MG_DAMP_RESTRICTION= 0.8
+%
+% Damping factor for the correction prolongation
+MG_DAMP_PROLONGATION= 0.8
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+% Convective numerical method (JST, LAX-FRIEDRICH, CUSP, ROE, AUSM, HLLC,
+%                              TURKEL_PREC, MSW)
+CONV_NUM_METHOD_FLOW= ROE
+%
+% Monotonic Upwind Scheme for Conservation Laws (TVD) in the flow equations.
+%           Required for 2nd order upwind schemes (NO, YES)
+MUSCL_FLOW= YES
+%
+% Slope limiter (NONE, VENKATAKRISHNAN, VENKATAKRISHNAN_WANG,
+%                BARTH_JESPERSEN, VAN_ALBADA_EDGE)
+SLOPE_LIMITER_FLOW= NONE
+%
+% 2nd and 4th order artificial dissipation coefficients
+JST_SENSOR_COEFF= ( 0.5, 0.02 )
+%
+% Time discretization (RUNGE-KUTTA_EXPLICIT, EULER_IMPLICIT, EULER_EXPLICIT)
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+
+% -------------------- TURBULENT NUMERICAL METHOD DEFINITION ------------------%
+%
+% Convective numerical method (SCALAR_UPWIND)
+CONV_NUM_METHOD_TURB= SCALAR_UPWIND
+%
+% Monotonic Upwind Scheme for Conservation Laws (TVD) in the turbulence equations.
+%           Required for 2nd order upwind schemes (NO, YES)
+MUSCL_TURB= NO
+%
+% Slope limiter (VENKATAKRISHNAN, MINMOD)
+SLOPE_LIMITER_TURB= VENKATAKRISHNAN
+%
+% Time discretization (EULER_IMPLICIT)
```

Can you cleanup the unused/default options please? Take a look at our other config files to see what I mean.
```suggestion
MUSCL_TURB= NO
```